Sources/SwiftSyntaxBuilder/generated/SyntaxExpressibleByStringInterpolationConformances.swift

```diff
@@ -19,8 +19,10 @@ import SwiftParser
 import SwiftParserDiagnostics
 
 extension SyntaxParseable {
-  public init(stringInterpolationOrThrow stringInterpolation: SyntaxStringInterpolation) throws {
-    self = try performParse(source: stringInterpolation.sourceText, parse: { parser in 
+  public typealias StringInterpolation = SyntaxStringInterpolation
```

Comment:
Why wasn't this needed before?

Reply:
I honestly don’t know and gave up trying to figure out why we need it now after a couple of minutes.